snippets/csharp/VS_Snippets_CLR_System/system.guid.ctor/cs/ctor1.cs

```diff
@@ -13,24 +13,25 @@ public static void Main()
       foreach (var guidString in guidStrings) {
          Guid guid = new Guid(guidString);
          Console.WriteLine("Original string: {0}", guidString);
```

Comment:
if we're adding string interpolation to the examples, they should be done consistently:
```suggestion
         Console.WriteLine($"Original string: {guidString}");
```